src/Symfony/Component/Routing/Tests/RouteTest.php

```diff
@@ -264,22 +264,4 @@ public function testSerializeWhenCompiled()
         $this->assertEquals($route, $unserialized);
         $this->assertNotSame($route, $unserialized);
     }
-
-    /**
-     * Tests that the serialized representation of a route in one symfony version
-     * also works in later symfony versions, i.e. the unserialized route is in the
-     * same state as another, semantically equivalent, route.
-     */
-    public function testSerializedRepresentationKeepsWorking()
```

Comment:
I'm not sure this test should be removed entirely.

Reply:
@stof – yes, please see Tobias' remark and my response above.